setup.py

```diff
@@ -69,6 +69,9 @@
             =======================
 
             See http://jacquev6.github.com/PyGithub"""),
+        install_requires=[
+          "requests==2.9.1"
```

Comment:
I just like fixing versions so nothing breaks on me unexpectedly.  Probably fine to remove the version alltogether.